packages/flutter_tools/lib/src/web/bootstrap.dart

```diff
@@ -158,7 +158,7 @@ String generateMainModule({
   String bootstrapModule = 'main_module.bootstrap',
 }) {
   return '''
-/* ENTRYPOINT_EXTENTION_MARKER */
+/* ENTRYPOINT_EXTENSION_MARKER */
```

Comment:
Looks like this typo is load-baring, `package:build` relies on the incorrect spelling:

https://github.com/dart-lang/build/blob/cbf5b2aa96f47aac823117548c1fc6f272a12ba5/build_web_compilers/lib/src/dev_compiler_bootstrap.dart#L492

Let's revert this change, we'd have to get the typo fixed upstream in `package:build` first.

Edit: looks like they have declined previously to fix this, see https://github.com/dart-lang/build/issues/3360.